README.md

```diff
@@ -1,71 +1,52 @@
-# Getting Started with Create React App
+### BookStore ###
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+My Bookstore app is a website to show a list of books using a given design and using an API, React, Redux, Javascript and Css to build and style it.
 
-## Available Scripts
+## Built With
 
-In the project directory, you can run:
+# HTML, CSS, JavaScript
+# React, Redux
 
-### `npm start`
+## Live Demo
+not yet...
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Getting started
+To get a local copy up and running follow these simple example steps. (NPM)
```

Comment:
- [ ] I think the README file is updated so kindly update it to the provided template from microverse you can check this link and please update all the sections to make it descriptive and professional 🚀✨💯🔥 [Readme](https://github.com/microverseinc/readme-template)

Reply:
and my readme is updated just not on the dev or master branches. But of course it will be merged